toolkit/scripts/check_static_glibc.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/python3
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+from pyrpm.spec import Spec
+from os.path import dirname, realpath
+
+import argparse
+import re
+import sys
+
+def get_glibc_version():
+    glibc_spec = Spec.from_file("SPECS/glibc/glibc.spec")
+    epoch_prefix = "{}:".format(glibc_spec.epoch) if glibc_spec.epoch else ""
+    release_without_dist = glibc_spec.release.replace("%{?dist}", "")
+    release_suffix = "-{}".format(release_without_dist)
```

Comment:
For consistency (and to keep things short), could we use the f-strings everywhere:

```suggestion
    glibc_spec = Spec.from_file("SPECS/glibc/glibc.spec")
    epoch_prefix = f"{glibc_spec.epoch}:" if glibc_spec.epoch else ""
    release_without_dist = glibc_spec.release.replace("%{?dist}", "")
    release_suffix = f"-{release_without_dist}"
```